packages/react-vapor/src/components/editor/CodeEditor.tsx

```diff
@@ -86,12 +86,16 @@ export class CodeEditor extends React.Component<ICodeEditorProps, CodeEditorStat
                 }}
                 value={this.state.value}
                 onChange={(editor, data, value: string) => this.handleChange(value)}
-                options={_.extend({}, CodeEditor.Options, {readOnly: this.props.readOnly, mode: this.props.mode})}
+                options={_.extend({}, CodeEditor.Options, {readOnly: this.removeCursorWhenEditorIsReadOnly(this.props.readOnly), mode: this.props.mode})}
                 className={this.props.className}
             />
         );
     }
 
+    private removeCursorWhenEditorIsReadOnly(isItReadOnly: boolean) {
+        return isItReadOnly === true ? 'nocursor' : this.props.readOnly;
+    }

```

Comment:
You can remove the function parameter and simply use `this.props.readOnly` instead of `isItReadOnly `

Reply:
You could remove the `=== true` because `isItReadOnly` is already a boolean.